src/components/Tooltip/tooltipPropTypes.js

```diff
@@ -24,6 +26,12 @@ const propTypes = {
     /** Any additional amount to manually adjust the vertical position of the tooltip.
     A positive value shifts the tooltip down, and a negative value shifts it up. */
     shiftVertical: PropTypes.oneOfType([PropTypes.number, PropTypes.func]),
+
+    /** Number of pixels to set max-width on tooltip  */
+    maxWidth: PropTypes.number,
+
+    /** Maximum number of lines to set on tooltip */
```

Comment:
`numberOfLines` is the maximum number of lines the tooltip will show, right? If so I think this is a bit clearer

```suggestion
    /** Maximum number of lines to show in tooltip */
```

Reply:
Done.